src/commonMain/kotlin/org/cru/godtools/tool/state/State.kt

```diff
@@ -0,0 +1,23 @@
+package org.cru.godtools.tool.state
+
+import org.cru.godtools.tool.internal.parcel.Parcelable
+import org.cru.godtools.tool.internal.parcel.Parcelize
+import org.cru.godtools.tool.model.EVENT_NAMESPACE_STATE
+import org.cru.godtools.tool.model.EventId
+
+@Parcelize
```

Comment:
I believe this is coming from the file above where you do ```expect annotation class Parcelize()```, is this just for linting?

Reply:
yes this is coming from the `expect`/`actual` annotation definition.

In this case we wanted to utilize the [Kotlin Parcelize](https://developer.android.com/kotlin/parcelize) gradle plugin to automatically make this object `Parcelable` for usage on Android (this annotation and the corresponding interface that `State` uses are no-ops on iOS and JS).

There were several options as to how to implement the Android specific logic, this seemed like the cleanest choice currently. The other option I looked at was creating an `AndroidState` subclass for Android only, and then use an `expect`/`actual` method to create a new `State` object. Depending on how requirements shift in the future we can switch to other models as needed.